crates/nu-cli/src/shell/filesystem_shell.rs

```diff
@@ -865,8 +865,8 @@ pub(crate) fn dir_entry_dict(
         #[cfg(unix)]
         {
             for column in [
-                "name", "type", "target", "readonly", "mode", "uid", "group", "size", "created",
-                "accessed", "modified",
+                "name", "type", "target", "links", "readonly", "mode", "uid", "group", "size",
```

Comment:
nit: I wonder if we should call this `num_links` or `link_count`. `links` makes me think I'll have a list of all the links. What do you think?

Reply:
I agree, just pushed an update to rename as `num_links`.